app/server/src/routes/user/user.controller.js

```diff
@@ -0,0 +1,152 @@
+const UserModel = require("../../models/user/user.model");
+const auth = require("../../services/auth");
+const jwt = require("jsonwebtoken")
+//const {authorization_admin} = require("../../services/auth");
+//const axios = require("axios");
+const jwt_ac_secret = process.env.JWT_AC_KEY
+const jwt_ref_secret = process.env.JWT_REF_KEY
+const access_jwtExpiry = '24h'
```

Comment:
Instead of giving 24hours, i think we should decrease this to about 6-8 hours. That could be more secure, in the sense that access tokesn would be refreshed more frequently... Also, maybe reduce refresh expiry to 5-7 days

Reply:
Done!